homeassistant/components/teslemetry/strings.json

```diff
@@ -470,6 +470,141 @@
     },
     "invalid_cop_temp": {
       "message": "Cabin overheat protection does not support that temperature"
+    },
+    "set_scheduled_charging_time": {
+      "message": "Time required to complete the operation"
+    },
+    "set_scheduled_departure_preconditioning": {
+      "message": "Departure time required to enable preconditioning"
+    },
+    "set_scheduled_departure_off_peak": {
+      "message": "End off peak time required to enable off peak charging"
+    }
```

Comment:
Sounds to me they should be required by the schema then and unnecessary to handle separately?

Reply:
But it's conditionally required, you only have to specify a time if you enable that feature. I can't do that with the schema AFAIK.

Reply:
Ok. Let me take a look then 👍

Reply:
Could we rephrase it slightly so the user understands the "condition" required.

Reply:
I have tried to reword this using the name from the service translation.